suse_openstack_cloud

```diff
@@ -395,14 +395,14 @@ fi
 ###############################################################
 section_header "RabbitMQ broker"
 
-if [ -e /usr/sbin/rabbitmqctl ]; then
-    rabbitmq_tmpdir=$(mktemp -d)
-    rabbitmq_report_log=$rabbitmq_tmpdir/rabbitmq_report.log
+rabbitmqctl=/usr/lib64/rabbitmq/bin/rabbitmqctl
```

Comment:
ah, ok, so its because supportutils overwrites CMDLINE with crap, and the rabbitmqctl script does only append. as vuntz already said, just running rabbitmqctl script as rabbitmq user fixes the issue already. We shouldn't rely on implementation details here.